LLNL/spack
# spack_cmd: Find external python

Spack_cmd.bat no longer requires configuration, and will find python and spack on its own. It preferentially tries to use python on your PATH

To test:
- Run this version of spack_cmd.bat and verify it locates python & spack from spackinstdir
- `spack make-installer -s [this version of spack] [build dir]`
install and verify spack_cmd.bat still lands in the correct directory
